imports/message-read-receipt/server/lib/ReadReceipt.js

```diff
@@ -50,11 +51,12 @@ export const ReadReceipt = {
 		if (message.unread && message.ts < firstSubscription.ls) {
 			Messages.setAsReadById(message._id, firstSubscription.ls);
 		}
+		const extraData = roomTypes.getConfig('l').getReadReceiptsExtraData(message);
```

Comment:
He discussed about it last week:
```suggestion
		const extraData = roomTypes.getConfig(room.t).getReadReceiptsExtraData(message);
```

You need to find the room and get the room type field:
const room = Rooms.findOneById(roomId, { fields: { t: 1 } });